### ISSUE TITLE
SlaveInstanceLaunchConfig Connection Error

### ISSUE THREAD

**Issue by TakaraShinya** [classmethod-aws/fullbok#13]
error occured jmeter slave instance /var/log/cfn-init.log
template is attach files

```
2016-09-06 07:11:24,853 [DEBUG] Describing resource SlaveInstanceLaunchConfig in stack arn:aws:cloudformation:ap-northeast-1:XXXXXXXXXX:stack/fullbok-demo 2016-09-06 07:11:24,868 [ERROR] ConnectionError
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/cfnbootstrap/util.py", line 162, in _retry
    return f(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/cfnbootstrap/util.py", line 234, in _timeout
    raise exc[0]
ConnectionError: ('Connection aborted.', gaierror(-3, 'Temporary failure in name resolution'))
2016-09-06 07:11:24,869 [DEBUG] Sleeping for 0.839813 seconds before retrying
2016-09-06 07:11:25,715 [DEBUG] Describing resource SlaveInstanceLaunchConfig in stack arn:aws:cloudformation:ap-northeast-1:XXXXXXXXXX:stack/fullbok-demo 2016-09-06 07:11:25,865 [ERROR] ConnectionError
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/cfnbootstrap/util.py", line 162, in _retry
    return f(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/cfnbootstrap/util.py", line 234, in _timeout
    raise exc[0]
```

**Comment by TakaraShinya** [comment]
[fullbok-demo.template.txt](https://github.com/classmethod-aws/fullbok/files/456525/fullbok-demo.template.txt)
[fullbok.template.txt](https://github.com/classmethod-aws/fullbok/files/456524/fullbok.template.txt)

**Comment by noto-satoshi** [comment]
@TakaraShinya, this issue resolved by #15. Sorry for taking long to write you back.
